packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -296,14 +315,24 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({
           }
         })
         .then(() => {
-          if (data.image) {
-            return createAttachment(patientUuid, data.image);
+          if (images && images.length > 0) {
+            return Promise.all(
+              images.map((image) => {
+                const uploadedFile: UploadedFile = {
+                  base64Content: image.base64Content,
+                  fileName: 'filename',
+                  fileType: image.fileType,
+                  fileDescription: 'description',
```

Comment:
It would be good to collect this from the user somehow or omit it altogether rather than supplying junk data.